transport-rxtx/src/main/java/io/netty/channel/rxtx/package-info.java

```diff
@@ -16,5 +16,8 @@
 
 /**
  * A serial and parallel port communication transport based on <a href="http://rxtx.qbang.org/">RXTX</a>.
+ *
+ * @deprecated this transport will be removed in the next major version.
  */
+@Deprecated
```

Comment:
Compiler in IntelliJ IDEA still is not allow `@Deprecated` in package-info's:
> Error:(23, 1) java: modifier deprecated not allowed here

Reply:
Fixed it... ff592c8d5c54d6a81b17050f980c5d57299a5148